Build failure: depends on `num`

~~~
CAMLC -c src/munify.mli
File "src/munify.mli", line 6, characters 18-33:
Error: Unbound module Big_int
make[1]: *** [Makefile:584: src/munify.cmi] Error 2
~~~

NB: the `num` library is installed, but the build-system does not look for it.

--- a/.merlin
+++ b/.merlin
@@ -1,6 +1,0 @@
-FLG -rectypes -thread -w @1..50@59-4 -safe-string
-
-S src
-B src
-
-PKG threads threads.posix coq.intf coq.ltac coq.idetop

--- a/Makefile.local
+++ b/Makefile.local
@@ -1,0 +1,4 @@
+CAMLPKGS += -package num
+
+merlin-hook::
+	$(HIDE)echo 'PKG num' >> .merlin

